consensus/src/commit_notifier.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::QuorumStoreError;
+use anyhow::{format_err, Result};
+use aptos_infallible::Mutex;
+use aptos_metrics::monitor;
+use consensus_types::{common::Round, request_response::ConsensusRequest};
+use futures::channel::{mpsc, mpsc::Sender, oneshot};
+use std::time::Duration;
+use tokio::time::timeout;
+
+/// Notification of consensus committed logical time for QuorumStore to clean.
```

Comment:
I think it's execution committed time, which may be a significant difference since consensus is decoupled and consensus commit doesn't imply we can state-sync to a round.